src/background/services/outgoingPaymentGrant.ts

```diff
@@ -380,6 +382,19 @@ export class OutgoingPaymentGrantService {
     }
   }
 
+  private computeHash = async (
+    clientNonce: string,
+    interactionNonce: string,
+    interactRef: string,
+    authServer: string,
+  ) => {
+    const data = new TextEncoder().encode(
+      `${clientNonce}\n${interactionNonce}\n${interactRef}\n${ensureEnd(authServer, '/')}`,
```

Comment:
I don't follow why https://openpayments.dev/identity/hash-verification/#verifying-hash expects a trailing slash for authServer.

Reply:
Just in case:
```sh
$ curl 'https://ilp.interledger-test.dev/sid-mxn-1' | jq .
{
  "id": "https://ilp.interledger-test.dev/sid-mxn-1",
  "publicName": "MXN sid@ilf",
  "assetCode": "MXN",
  "assetScale": 2,
  "authServer": "https://auth.interledger-test.dev/f537937b-7016-481b-b655-9f0d1014822c",
  "resourceServer": "https://ilp.interledger-test.dev/f537937b-7016-481b-b655-9f0d1014822c",
  "cardService": "http://testnet-rafiki-cards-service:3007/"
}
```

Reply:
>  I don't follow why https://openpayments.dev/identity/hash-verification/#verifying-hash expects a trailing slash for authServer.

It's not necessary, I will put up a change in the docs